### DISCUSSION TITLE
Problem with Input() in 'Run your code' / 'Run Python code in your browser'

### DISCUSSION THREAD

**Discussion by antonthys** [dodona-edu/feedback-and-support#26] [Questions - Vragen] [answered]
Hi, 

A lot of my students are getting errors when testing their Python code in the '**Run your code**' / '**Run Python code in your browser**' editor when they use an Input() function. The codes keeps on running without asking the user for an input, or the user receives an error that this functionality is not working. For some students it works normal.

Since the students use Dodona for their exams, I'm using an MDM-software to whitelist dodona.be, blocking all other websites. Could this be the reason for this malfunctioning? Is there a way to resolve this problem? 

Many thanks in advance!

**Comment by jorg-vr** [comment, marked as answer]
Hi, thanks for notifying us of this issue.

On our end we also aren't experiencing any issues, so it is hard for us to reproduce the issue.

The following information could help me reproduce the problem:
- The browser version
- A screenshot of the error
- A screenshot of the browser console output. This link shows how to open the browser console for different browsers: https://appuals.com/open-browser-console/

You could also try if any of the following things solve the problem:
- Using the editor without MDM active
- Disabling all browser extensions
- Updating the browser to the newest version
- Using a different browser
